README.md

```diff
@@ -1 +1,229 @@
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Agro-preneurship summit] <a name="about-project"></a>
+
+> This project is website about an Agro-preneurship summit holding in Nigeria. it's a training program for practising and aspiring farmers
+
+**[your_project__name]** is a...
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+> Html, css and javascript.
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://www.learn-html.org/">html</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://web.dev/learn/css/">css</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.javascript.com/">Javascript</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+> This project include some level of animation, it is also accesible for people with disabilities and it's responsive to any device.
+
+- **[Animations]**
+- **[Accesibility]**
+- **[Responsiveness]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> This is the link to my live Demo
+
+- [Live Demo Link](https://divinity-dev.github.io/Capstone-project/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+<!-- VIDEO  PRESENTATION -->
+
+## 🚀 Video presentation <a name="video-demo"></a>
+
+> This is the link to my live Demo
+- [Video Link](https://www.loom.com/share/ea538914aead42af99c3eb36858ebd2c)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> The project is very easy to use, all that's required is to visit the site link or for developers that might want to use the work, just head over to my github for the codes
+
+### Prerequisites
+
+All yoou need to run the code is a browser
+
+
+
+
+To run the project, execute the following command:
+
+<!--
+Example command:
+
+```sh
+  rails server
+```
+--->
+
+### Run tests
+
+There are no commands required to run the code
+
+<!--
+Example command:
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+> Asiriuwa O. Divine
+
+👤 **Author1**
+
+- GitHub: [@githubhandle](https://github.com/Divinity-dev/)
+- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/divine-asiriuwa-a87227a3/)
+
+👤 **Author2**
+
+- GitHub: [@githubhandle](https://github.com/Divinity-dev/)
+- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
```

Comment:
[REQUIRED] As suggested by the previous reviewer kindly remove the `Author 2` section and leave only `Author 1`  since you are the only one who authored this project.